metatensor-core/CMakeLists.txt

```diff
@@ -409,6 +409,7 @@ write_basic_package_version_file(
 install(FILES "include/metatensor.h" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 install(FILES "include/metatensor.hpp" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 install(DIRECTORY "include/metatensor" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+install(DIRECTORY "include/dlpack" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
```

Comment:
So I don't think we should install there, we don't want to override existing dlpack headers. 

Are we sure we want the end user to `#include <dlpack/dlpack.h>`, or should they do `#include <metatensor/dlpack.h>` for full clarity on where the header comes from?